src/dev/konnector.md

```diff
@@ -39,179 +56,168 @@ yarn # or npm install
 
 _note: the Cozy Team uses `yarn`, but if you prefer `npm`, just keep using it, everything should just work._
 
-Then, write your code into `index.js` and build the application, running `yarn build` or `npm run build`.
+The connector template already comes with some functionnal and commented code which scrapes
+the [books.toscrape.com](http://books.toscrape.com) website, for which you do not need to have
+credentials.
+
+As indicated in the README, just run it:
+
+```sh
+yarn standalone
+```
+
+The first run will create a `konnector-dev-config.json` file which allows you to configure the input
+of the connector when running it in the CLI.
+
+```javascript
+{
+  "COZY_URL": "http://cozy.tools:8080",
+  "fields": {}
+}
+```
+
+`COZY_URL` is for later, but the `fields` attribute will allow you to define credentials to the
+target web service, like `login` and `password` as if they would come from a real cozy.
+
+This way to run the connector is the *standalone* mode. In this mode, `cozyClient` is stubbed and
+all data meant to be saved in a cozy is displayed in the standard output and files are directly
+saved in the root directory of the connector. This is usefull to first develop your connector
+without handling the state of a real cozy.
+
+You have more documentation about this int the [CLI section of the documentation](https://github.com/cozy/cozy-konnector-libs/blob/master/docs/cli.md).
```

Comment:
Un "t" en trop dans *about this *in* the"

Reply:
s/usefull/useful